File: Semester 6/Android/Lab1/app/src/main/java/com/example/myapplication/MainActivity.java
package com.example.myapplication;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    public void SetRedBackground(View view) {
        view.getRootView().setBackground(getResources().getDrawable(R.color.red));
    }

    public void SetYellowBackground(View view) {

        view.getRootView().setBackground(getResources().getDrawable(R.color.yellow));
    }

    public void SetGreenBackground(View view) {
        view.getRootView().setBackground(getResources().getDrawable(R.color.green));

    }
}

File: Semester 6/Android/kotlin/Lab1/app/src/main/java/com/example/lab1/MainActivity.kt
package com.example.lab1

import android.graphics.Color.red
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }

    fun SetRedBackground(view: View) {
        view.rootView.setBackgroundColor(resources.getColor(R.color.red));
    }

    fun SetYellowBackground(view: View) {
        view.rootView.setBackgroundColor(resources.getColor(R.color.yellow));
    }

    fun SetGreenBackground(view: View) {
        view.rootView.setBackgroundColor(resources.getColor(R.color.green));
    }

}
